code/LeftAndMain.php

```diff
@@ -1757,6 +1761,46 @@ public function SiteConfig()
         return class_exists(SiteConfig::class) ? SiteConfig::current_site_config() : null;
     }
 
+    /**
+     * The urls used for the links in the Help dropdown in the backend
+     *
+     * @config
+     * @var array
+     */
+    private static $help_links = [
+        'CMS User help' => 'https://userhelp.silverstripe.org/en/4',
```

Comment:
Spoke offline—If we moved to the flat array structure it would be a lot more difficult for developers to update the default help menu links via .yml so will leave array structure as is.

Reply:
Without the silverstripe-cms the CMS User help link is still helpful to users due to admin sections like 'Security' (which exists in the admin module) and would appear in the CMS menu list. User help such as [adding new members](https://userhelp.silverstripe.org/en/4/managing_your_website/changing_and_managing_users/)  for example would be useful.